lib/millstone.js

```diff
@@ -499,15 +504,18 @@ function unzip(file, callback) {
             );
             const content = zipFile.getData();
             const overwrite = true;
-            writeFileToAsync(dest, content, overwrite, (written) => {
+            writeFileToAsync(dest, content, overwrite, (err, written) => {
+                if (err) {
+                    return callback(err);
+                }
                 if (written) {
                     remain--;
                     if (!remain) {
                         add_item_to_metafile(metafile,'unzipped_file',new_name,function(err) {
                             // ignore error from add_item_to_metafile
                             //if (err && env == 'development') console.error('[millstone] ' + err.message);
                             if (err) throw err;
```

Comment:
I'm aware it's out of the scope of this PR, but please fix this sync error throw that raises an UncaughtException.
```suggestion
                            if (err) {
                                return callback(err);
                            };
```